src/rustllvm/PassWrapper.cpp

```diff
@@ -188,7 +188,7 @@ LLVMRustCreateTargetMachine(const char *triple,
     }
 
     TargetOptions Options;
-    Options.PositionIndependentExecutable = PositionIndependentExecutable;
+    //Options.PositionIndependentExecutable = PositionIndependentExecutable;
```

Comment:
This was [deleted here](http://reviews.llvm.org/D19753) which appears to be [replaced by this](http://reviews.llvm.org/D19671) which is a new `setPIELevel` function on the LLVM module itself.

The LLVM module isn't available at this time so we'll just have to arrange for it to be set later. Perhaps something like this though:

* Leave this block, but gated by `#if LLVM_VERSION_MINOR <= 8` for compatibility with older versions.
* Add `LLVMRustSetModulePIELevel` as a function which is a noop on pre-LLVM-3.9 and afterwards calls the `setPIELevel` function on the LLVM module.